psalm-baseline.xml

```diff
@@ -207,12 +207,9 @@
     </TypeDoesNotContainType>
   </file>
   <file src="src/DateTimeFormatter.php">
-    <PossiblyInvalidArgument>
-      <code><![CDATA[$e->getCode()]]></code>
-    </PossiblyInvalidArgument>
-    <MixedArgumentTypeCoercion>
-      <code><![CDATA[$value]]></code>
-    </MixedArgumentTypeCoercion>
+    <PossiblyInvalidMethodCall>
+      <code><![CDATA[format]]></code>
+    </PossiblyInvalidMethodCall>
```

Comment:
This shouldn't be in the baseline… Where is the InvalidMethodCall?

Reply:
the `InvalidMethodCall` came from the last line where we format the date. `$value->format($this->format);` .Psalm saw that at that point the `$vaue` could still be an `int`. I refactored a bit that logic and i could get rid io that baseline.